src/data/sample-question-interactive-complex.json

```diff
@@ -0,0 +1 @@
ractives/branch/interactive-lightbox/open-response/","click_to_play":false,"click_to_play_prompt":null,"description":"","enable_learner_state":true,"full_window":false,"has_report_url":false,"image_url":null,"name":"Open Response Interactive","native_height":435,"native_width":576,"no_snapshots":false,"show_delete_data_button":false,"thumbnail_url":"","customizable":true,"authorable":true}},"type":"ManagedInteractive","ref_id":"24-ManagedInteractive","linked_interactives":[]},"section":"interactive_box"}]},{"additional_sections":{},"embeddable_display_mode":"stacked","is_completion":false,"is_hidden":false,"layout":"l-6040","name":"Image Question","position":10,"show_header":false,"show_info_assessment":true,"show_interactive":true,"show_sidebar":false,"sidebar":null,"sidebar_title":"Did you know?","toggle_info_assessment":true,"embeddables":[{"embeddable":{"content":"\u003Cp\u003Etest content\u003C/p\u003E","is_full_width":false,"is_hidden":false,"name":"This is a text box thing","type":"Embeddable::Xhtml","ref_id":"5-Embeddable::Xhtml"},"section":null},{"embeddable":{"name":"","url_fragment":null,"authored_state":"{\"version\":1,\"questionType\":\"iframe_interactive\",\"imageFit\":\"shrinkBackgroundToCanvas\",\"imagePosition\":\"center\",\"stampCollection\":{\"collection\":\"molecules\"},\"stampCollections\":[{\"collection\":\"molecules\"}],\"prompt\":\"Draw something!\",\"required\":true,\"predictionFeedback\":\"\u003Cp\u003E\u003C/p\u003E\",\"defaultAnswer\":\"Now write about what you did\",\"hint\":\"\u003Cp\u003EThis is a \u003Cstrong\u003Ehint! \u003C/strong\u003E\u003Cem\u003Eeh?\u003C/em\u003E\u003C/p\u003E\",\"answerPrompt\":\"Why did you draw that?\"}","is_hidden":false,"is_full_width":true,"show_in_featured_question_report":true,"inherit_aspect_ratio_method":true,"custom_aspect_ratio_method":null,"inherit_native_width":true,"custom_native_width":576,"inherit_native_height":true,"custom_native_height":435,"inherit_click_to_play":true,"custom_click_to_play":false,"inherit_full_window":true,"custom_full_window":false,"inherit_click_to_play_prompt":true,"custom_click_to_play_prompt":null,"inherit_image_url":true,"custom_image_url":null,"library_interactive":{"hash":"a5d53c7ce6f9ec9a143fd06bff9108b42f3ff7ff","data":{"aspect_ratio_method":"DEFAULT","authoring_guidance":"","base_url":"https://models-resources.concord.org/question-interactives/branch/interactive-lightbox/image-question","click_to_play":false,"click_to_play_prompt":null,"description":"Image Question","enable_learner_state":true,"full_window":false,"has_report_url":false,"image_url":null,"name":"Image Question","native_height":435,"native_width":576,"no_snapshots":false,"show_delete_data_button":true,"thumbnail_url":"","customizable":false,"authorable":true}},"type":"ManagedInteractive","ref_id":"23-ManagedInteractive","linked_interactives":[]},"section":"interactive_box"}]}],"plugins":[],"type":"LightweightActivity","export_site":"Lightweight Activities Runtime and Authoring"}
```

Comment:
I've been using https://jsonformatter.org/ to format the single line JSON that we get when exporting an activity into something more readable.  It's helpful when reading the activity structure from the JSON sample files.

Reply:
Also the Fix JSON extension VS Code that you can instal from Preferences -> Extensions.

Then on any file, Shift-Cmd-P, hit Fix JSON and it will nicely format it.

Reply:
Nice extension!  Even easier!